tools/package-tools/entry-point-package-json.ts

```diff
@@ -8,8 +8,8 @@ export function createEntryPointPackageJson(destDir: string, packageName: string
     name: `@angular/${packageName}/${entryPointName}`,
     typings: `../${entryPointName}.d.ts`,
     main: `../bundles/${packageName}-${entryPointName}.umd.js`,
-    module: `../@angular/${packageName}/${entryPointName}.es5.js`,
-    es2015: `../@angular/${packageName}/${entryPointName}.js`,
+    module: `../esm5/${packageName}/${entryPointName}.es5.js`,
+    es2015: `../esm2015/${packageName}/${entryPointName}.js`,
```

Comment:
Out of curiosity: How does this work with Closure Compiler now? Is there an alternative to the `js_module_roots` option?

Reply:
Short answer: it doesn't.

I updated the script and left a comment explaining the current situation w/ closure.